test/unit/gateways/ixopay_test.rb

```diff
@@ -77,9 +82,26 @@ def test_failed_authorize
     assert_equal '2003', response.error_code
   end
 
-  def test_successful_capture; end
+  def test_successful_capture
+    @gateway.expects(:ssl_post).returns(successful_capture_response)
 
-  def test_failed_capture; end
+    response = @gateway.capture(@amount, 'auth-123')
+
+    assert_success response
+    assert_equal 'FINISHED', response.message
+    assert_equal '17dd1e0b09221e9db038|20191031-17dd1e0b09221e9db038', response.authorization
+    assert response.test?
+  end
+
+  def test_failed_capture
+    @gateway.expects(:ssl_post).returns(failed_capture_response)
+
+    response = @gateway.capture(@amount, 'auth-123')
```

Comment:
Consider changing the authorization value here to `nil` or something that more clearly illustrates that the value is invalid.

Reply:
Done.